src/PrestaShopBundle/Controller/Api/TranslationController.php

```diff
@@ -170,8 +170,8 @@ public function translationEditAction(Request $request)
             $translationService = $this->container->get('prestashop.service.translation');
             $response = [];
             foreach ($translations as $translation) {
-                if (!array_key_exists('theme', $translation)) {
-                    $translation['theme'] = null;
+                if (!array_key_exists('theme', $translation) || empty($translation['theme'])) {
```

Comment:
```suggestion
                if (empty($translation['theme'])) {
```
array_key_exists is already done by empty function.